hudi-client/src/test/java/org/apache/hudi/metrics/TestHoodieMetrics.java

```diff
@@ -22,22 +22,22 @@
 import org.apache.hudi.config.HoodieWriteConfig;
 
 import com.codahale.metrics.Timer;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import java.util.Arrays;
 import java.util.Random;
 
 import static org.apache.hudi.metrics.Metrics.registerGauge;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 public class TestHoodieMetrics {
```

Comment:
@yanghua @vinothchandar Some test classes do not have javadoc, while some others have. I think we'd better align on this just to standardize the codebase. If we agree that javadoc should be added to all test classes, then we'll fix them while making changes on those tests from now on.

Reply:
I agree we'd better have javadocs.